### PR TITLE
Add Corey to will

### PR DESCRIPTION
**Pull request by probablycorey**
It's only fair.

### CODE REVIEW

File: docs/will.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Default on any debt that is possible.
 * My pet cat, Giro (Giraud), goes to Jenn Chan. If Jenn is unable or unwilling to take care of said kitty cat, Lydia Katsamberis is willed the cat.
 * All Bitcoin, Dogecoin, and any other cryptocurrencies will be transfered to addresses provided by Yaroslav Shirokov (@sshirokov) unless he is to blame for my death. Then they will be directly transfered to the current mayor of San Francisco, of course if the candidate is not the deceased. http://skalnikformayor.com/
 * I will all of my unexercised Github stock options to Naomi Plasterer.
+* All my possions residing on my desk at GitHub HQ are bequeathed to Corey Johnson.
```
Comment (cheshire137) [inline docs/will.md]: > All my possions

Can I get one of these possions? I've never seen a possion in real life.

**Comment by probablycorey** [comment]
I can't handle PRs left unresolved, so I'm closing this!

**Comment by cheshire137** [comment]
He's dead, Jim.

**Comment by skalnik** [comment]
Shit, I missed out.

**Pull request closed without merging**